tests/baselines/reference/api/typescript.d.ts

```diff
@@ -2651,6 +2651,7 @@ declare namespace ts {
         interface ServerHost extends System {
             watchFile(path: string, callback: FileWatcherCallback, pollingInterval?: number, options?: WatchOptions): FileWatcher;
             watchDirectory(path: string, callback: DirectoryWatcherCallback, recursive?: boolean, options?: WatchOptions): FileWatcher;
+            preferNonRecursiveWatch?: boolean;
```

Comment:
Is the intent that downstream users configure this? Do we need to wait for David to check that this helps?

Reply:
I was not sure whether to leave this public on ServerHost or not but TscWatch yes because i think last i checked david was using the API to patch tsc watch.  

I am looking to get this in early and revert or reiterate sooner than later as we dont get more feedback without this being in nightly esp since this depends on OS and vscode setting to use their watchers

Reply:
i can make these internal on tsc-watch and tsserver internal and make them public in next PR if they are needed if that approach is better

Reply:
Understood; just checking; I think you have more context than me on this.